Aspire Budgeting/Views/Dashboard/DasboardCardsListView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  DasboardCardsListView.swift
+//  Aspire Budgeting
+//
+
+import SwiftUI
+
+struct DasboardCardsListView: View {
+
+  let cardViewItems: [DashboardCardView.DashboardCardItem]
+
+  var body: some View {
+    GeometryReader {g in
+      ScrollView {
+        VStack {
+          ForEach(0..<self.cardViewItems.count) { idx in
+            GeometryReader { geo in
+              BaseCardView<DashboardCardView>(
+//                       cardViewItem: self.cardViewItems[idx],
+                       minY: g.frame(in: .global).minY,
+                       curY: geo.frame(in: .global).minY,
+                baseColor: colorFor(idx: idx)) {
+                DashboardCardView(cardViewItem: self.cardViewItems[idx],
+                                  baseColor: colorFor(idx: idx))
+              }
+                .padding(.horizontal)
+            }.frame(maxWidth: .infinity)
+              .frame(height: 125)
+          }
+        }
+      }.background(Color.primaryBackgroundColor)
+    }
+  }
+}
+
+//MARK:- Internal Types
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/5d9f74b3b35e3fb9746b1333ffd07f18323321a5/Aspire Budgeting/Views/Dashboard/DasboardCardsListView.swift#L36"></span>MARK comment should be in valid format. e.g. ‘// MARK: …’ or ‘// MARK: - …’<br />
<code>mark</code> <code>DasboardCardsListView.swift:36</code></td>
    </tr>
  </tbody>
</table>